src/app/components/main/Main.tsx

```diff
@@ -0,0 +1,13 @@
+import * as Styled from './Main.styles';
+import { MainProps } from './Main.types';
+
+export const Main = ({ isScreenMobile, onFilterClick }: MainProps) => {
+  return (
+    <Styled.Main>
+      {isScreenMobile ? (
```

Comment:
Dwie rzeczy:
1. Można to zapisać bez ternary operator, zwykłym `&&`, jest krócej (to samo dotyczy `Sidebar`)
2. W linii 8. masz string, który powinien się znaleźć w translacji